docs/Concepts/4.md

```diff
@@ -1,60 +1,124 @@
 ---
-title: Security Features
+title: Discovery & Bootstrap Nodes
 ---
 
-Each protocol layer of Waku v2 provides a distinct service and is associated with a separate set of security features and concerns.
-Therefore, the overall security of Waku v2 depends on how the different layers are utilized.
+Node discovery is the mechanism that enables a Waku node to find other nodes.
 
-The security models are detailed in the RFC of the protocols.
-That we strive to provide well documented and open source RFCs so that Waku users know what security guarantees are and aren't provided by each protocol.
+When starting a Waku node, it needs to connect to other nodes to be able to send, receive and retrieve messages.
+This means there needs to be a discovery mechanism that enables finding other nodes when its initially not connected to any nodes.
+This process is called _bootstrapping_.
 
-Some of the security features of Waku v2 are:
+Once connected, the node needs to find additional peers to have:
 
-### Pseudonymity
+- Enough peers in the Waku Relay mesh (target is 6),
+- Enough peers in reserve, in case current peers are overloaded or go offline,
+- Peers with specific Waku capabilities (e.g. Store, Light Push, Filter).
 
-Waku v2 by default guarantees pseudonymity for all of the protocol layers since parties do not have to disclose their true identity
-and instead they utilize libp2p `PeerID` as their identifiers.
-While pseudonymity is an appealing security feature, it does not guarantee full anonymity since the actions taken under the same pseudonym
-i.e., `PeerID` can be linked together and potentially result in the re-identification of the true actor.
+## Default Bootstrap Mechanism
 
-### Anonymity / Unlinkability
+The default bootstrap mechanism is to connect to the Status' nwaku prod fleet.
```

Comment:
There is no default bootstrap mechanism from a Waku point of view, this section should be removed.